docs/Navigation/StopStream/index.md

```diff
@@ -9,6 +9,8 @@ Sender
 Purpose
 : Close the video streaming service between SDL and the HMI and stop streaming video data.
 
+The app must stop video streaming after receiving onHMIStatus with HMI level NONE or BACKGROUND. 
```

Comment:
I would like to say "The app must stop video streaming after receiving onHMIStatus with videoStreamingState=NOT_STREAMBLE", that covers the case for app goes to hml level NONE or BACKGROUND and LIMITED too.

Reply:
@yang1070 fixed 80ef5df